src/utils/uriToHttp.test.ts

```diff
@@ -1,7 +1,7 @@
 import uriToHttp from './uriToHttp'
 
 describe('uriToHttp', () => {
-  it('returns .eth.link for ens names', () => {
+  it('returns .eth.limo for ens names', () => {
```

Comment:
How does this pass?

Reply:
The 1st argument of `it` is the test's name.
Should have no impact on functionality. Also, that test in particular is comparing data from a place we shouldn't be able to reach resulting in an empty array.
I just did a replace all from `.eth.link` to `.eth.limo`.
I'd keep the change for consistency sake, but I can revert or even do something else with it...